packages/amplication-client/src/Workspaces/WorkspaceSelector.scss

```diff
@@ -84,33 +85,37 @@
     }
 
     &__button {
-      margin-left: var(--default-spacing);
-      margin-top: var(--default-spacing-small);
+      i {
+        padding: 2px;
```

Comment:
please use vars or params (I am not sure if we are using 2px anywhere so if needed, please create a new var)